### PR TITLE
SmartLocker small updates

### PR DESCRIPTION
**Pull request by gtlewis**

### CODE REVIEW

File: dapp/views/smartlocker/smartlocker.jsx [hunk 1]
```diff
@@ -20,7 +22,7 @@ const SmartLocker = ({ locker }) => {
     case 'smart':
       return <ManagementSubview/>
     default:
-      return <Bouncing/>
+      return <Bouncing class="bouncing"/>
```
Comment (tkali) [inline dapp/views/smartlocker/smartlocker.jsx]: `class` is not valid react parameter (`className` is). Why was this added? Bouncing already has its own class name.

Comment (tkali) [reply dapp/views/smartlocker/smartlocker.jsx]: OK, I see it corresponds with `.bouncing` class above. Changing styles of reusable, external components is frowned upon. I would rather wrap it in div:

```
<div className="loading-subview">
  <Bouncing>
</div>
```
and change `.bouncing` to `.loading-subview` accordingly. 

This place still needs some polish. Will do after demo.

File: dapp/views/smartlocker/subviews/create_or_link/create_or_link.jsx [hunk 2]
```diff
@@ -4,16 +4,18 @@ import { bindActionCreators }      from 'redux'
...
+import { flags }                   from 'lib/helpers'
 
 import './create_or_link.css'
 
 class CreateOrLinkSubview extends React.Component {
   constructor (props) {
     super(props);
-    // TODO: address of the registrar
-    // should be provided by the extension
-    this.registrar = new RegistrarContract(config.contracts.registrar);
```
Comment (tkali) [inline dapp/views/smartlocker/subviews/create_or_link/create_or_link.jsx]: :+1:  forgot about it, thanks!

File: dapp/views/smartlocker/subviews/create_or_link/create_or_link.jsx [hunk 3]
```diff
@@ -4,16 +4,18 @@ import { bindActionCreators }      from 'redux'
...
+
+    const { abi } = config.contracts.registrar;
+    const { address } = props.locker.registrar;
+
+    this.registrar = new RegistrarContract({ abi, address });
```
Comment (tkali) [inline dapp/views/smartlocker/subviews/create_or_link/create_or_link.jsx]: I think at this point calling `new RegistrarContract(abi, address)` would be nicer. That requires updating `contracts.js` with something like:

```
constructor(abi, address)
```
and 
```
super(abi, address)
```

File: dapp/views/smartlocker/smartlocker.jsx (hunk 1 above)
Comment (gtlewis) [reply dapp/views/smartlocker/smartlocker.jsx]: Done

File: dapp/views/smartlocker/subviews/create_or_link/create_or_link.jsx (hunk 3 above)
Comment (gtlewis) [reply dapp/views/smartlocker/subviews/create_or_link/create_or_link.jsx]: Done
